bridge/pier/common.go

```diff
@@ -85,6 +87,32 @@ func isProposer(cliCtx cliContext.CLIContext) bool {
 	return false
 }
 
+// IsStateSyncer returns if current user is state syncer or not
+func IsStateSyncer(cliCtx cliContext.CLIContext) bool {
+	var stateSyncerList []types.Validator
+	isStateSyncer := false
```

Comment:
We can remove this, because we are anyway implicitly returning `false` and `true`